### PR TITLE
Add hairline test.

### PR DESCRIPTION
**Pull request by strarsis**
This PR adds a test for CSS hairlines support in the browser.
I also found this blog post very useful for explaining CSS hairlines 
and how to test for support (test is derived from it): http://dieulot.net/css-retina-hairline

With best regards

### CODE REVIEW

File: test/browser/integration/caniuse.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ window.caniusecb = function(caniuse) {
       geolocation: 'geolocation',
       getrandomvalues: 'getrandomvalues',
       getusermedia: 'stream',
+      hairline: 'hairline',
```
Comment (patrickkettner) [inline test/browser/integration/caniuse.js]: I am not [seeing this on caniuse](http://caniuse.com/#search=hairline) - can you remove unlessI am missing something

File: feature-detects/css/hairline.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "tags": ["css"],
+  "authors": ["strarsis"],
+  "notes": [{
+    "name": "Blog post about CSS retina hairlines",
+    "href": "http://dieulot.net/css-retina-hairline"
+  },{
+    "name": "Derived from",
+    "href": "https://gist.github.com/dieulot/520a49463f6058fbc8d1"
+  }]
+}
```
Comment (patrickkettner) [inline feature-detects/css/hairline.js]: can you add a description as to what hairline is? I don't think the naming is super well known

File: feature-detects/css/hairline.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+!*/ define(['Modernizr', 'testStyles'], function(Modernizr, testStyles) {
+  Modernizr.addTest('hairline', function() {
+    return testStyles('#modernizr {border:.5px solid transparent;}', function(elem) {
```
Comment (patrickkettner) [inline feature-detects/css/hairline.js]: no need for a trailing semicolon

File: feature-detects/css/hairline.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      return elem.offsetHeight == 1;
```
Comment (patrickkettner) [inline feature-detects/css/hairline.js]: triple equals please

**Comment by patrickkettner** [comment]
couple of small nits, then lgtm. After you are done, could you rebase down to a single commit? Thanks!

**Comment by strarsis** [comment]
Fixed + squashed.

With best regards

**Pull request merged**